data_diff/diff_tables.py

```diff
@@ -239,12 +236,31 @@ def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_coun
         for res in self._thread_map(list, diff_iters):
             yield from res
 
+    def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_count=None):
+        logger.info(
+            ". " * level
+            + f"Diffing segment {segment_index}/{segment_count}, key-range: {table1.start_key:x}..{table2.end_key:x}"
```

Comment:
I think the size is really useful too, even if it's not actually the # of rows in some cases.

Reply:
Why you you writing these in hex form? Pretty hard to read

<img width="914" alt="CleanShot 2022-05-27 at 12 33 45@2x" src="https://user-images.githubusercontent.com/97400/170741270-a302c953-13c5-48f6-89fa-9bccc96701ba.png">